templates/htmx-survey.html

```diff
@@ -17,8 +17,10 @@
                 hx-post="/surveys/submit-survey/{{ survey_id }}"
                 hx-include="[name='question']"
                 hx-swap="innerHTML"
-                _="on click toggle .hidden on #survey">
+                _="on click toggle .hidden on #survey
+                and window.location.reload()">
                 {{_('survey_submit')}}</button>
```

Comment:
this will refresh the page. We don't want that. To fix it:
- extract the #survey_button to a separate htmx- component.
   - taking into account the other changes.
- then return it whenever a change is made within the survey.py
- this way it's reuseable and no page refresh is needed.

Reply:
Same here, if I do this and add it to survey.py, it won't be generic anymore and if I add it to htmx-survey.html also.... Or do you have a workaround for that?

Reply:
See how the `htmx-tags-list.html` works. This should be the same!

Reply:
yes I understand! But survey.py is generic and if I add the class button it won't be generic because then survey's would always be rendered with a button from now on, but if you think that is fine then I'll add it!

Reply:
> extract the #survey_button to a separate htmx- component.

By this i meant the following:
```html
<!-- htmx-survey-status.html -->
<h2 id="survey_status" class="flex ltr:ml-auto rtl:mr-auto">
  ...
</h2>
```

Then, you could just put this in the `class-overview.html` the same way i do with the tags-list! Then this component will update itself. This is how we will move to autosaving! If you don't know what I'm talking about, don't worry, will fix it in that PR. Your PR already does a lot :)